src/main/scala/cognite/spark/v1/FileContentRelation.scala

```diff
@@ -144,3 +144,7 @@ class FileContentRelation(config: RelationConfig, fileExternalId: String, inferS
     }
 
 }
+
+object FileContentRelation extends NamedRelation {
+  override val name: String = "filecontent"
```

Comment:
looks like we have a todo for file content - add datasource metrics like others, make it extend CdfRelation

Reply:
Ah yes, this is the companion object but you are right here :s

Reply:
Will do in another PR if that's ok

Reply:
Will do in this PR, sounds like it's easy.